backend/src/about/views.ts

```diff
@@ -11,13 +17,25 @@ process.env.NODE_ENV === "test"
   ? (useAuth = NoAuth as RequestHandler)
   : (useAuth = auth as RequestHandler);
 
-aboutRouter.get("/", useAuth, async (req: Request, res: Response) => {
-  attempt(res, 200, () => aboutController.getAboutPageContent());
-});
+aboutRouter.get(
+  "/",
+  (authIfAdmin as RequestHandler) ||
+    (authIfSupervisor as RequestHandler) ||
+    (authIfVolunteer as RequestHandler),

```

Comment:
I want to suggest creating a variable that already has the correct type, and then reusing it, similar to what is done for basic auth in event/views.ts instead of copying and pasting this

Reply:
Also, everyone should be able to get the about page content but only admins can edit. So this should be revised